terraform/shared/main.tf

```diff
@@ -0,0 +1,18 @@
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = "rg-shared-gitops-demo"
+  location = "Central US"
```

Comment:
```suggestion
  location = "westeurope"
```
Better latency

Reply:
Money. I'm using my personal account with a few euros for the first month 🤷